tests/benchmarks/test_join.py

```diff
@@ -1,52 +1,51 @@
+import dask
 import dask.dataframe as dd
 import pytest
 
 from ..utils_test import cluster_memory, run_up_to_nthreads, timeseries_of_size
 
-mem_mult = [
-    0.1,
-    pytest.param(
-        1,
-        marks=pytest.mark.skip(reason="Does not finish"),
-    ),
-    pytest.param(
-        10,
-        marks=pytest.mark.skip(reason="Does not finish"),
-    ),
+params = [
```

Comment:
Can we put a comment that says what these params are?  I know you can read it in the `pytest.mark()` but it helps, before it was clear because it was just `mem_mult`, something like
```suggestion
# (mem_mult, shuffle)
params = [
```